docs/build_version_doc/build_all_version.sh

```diff
@@ -21,7 +21,7 @@
 # Built files are stored in $built
 # Version numbers are stored in $tag_list.
 # Version numbers are ordered from latest to old and final one is master.
-tag_list="0.11.0.rc3 master"
+tag_list="0.12.0 0.11.0 master"
```

Comment:
@sandeep-krishnamurthy @kevinthesun 
Should this change be made now or after the RC is final?